.buildkite/scripts/run_models.sh

```diff
@@ -19,4 +19,6 @@ dbt deps
 dbt seed --target "$db" --full-refresh
 dbt run --target "$db" --full-refresh
 dbt test --target "$db"
+dbt run --vars '{full_statement_version: false}' --target "$db" 
```

Comment:
`full_statement_version` is not a variable, but rather an argument to a particular macro. This ultimately is defining and setting a new variable that is not used. We can remove this.
```suggestion
```

Reply:
Instead, you will want to update [models themselves](https://github.com/fivetran/dbt_salesforce_formula_utils/tree/main/integration_tests/models) within the integration tests folder to have this `full_statement_version: false` argument declared.

I would recommend adding this to any models that do not already have `full_statement_version: true`. This way we still test the `true` and `false` version.

For example, I **would not recommend changing** [account_statement_view](https://github.com/fivetran/dbt_salesforce_formula_utils/blob/main/integration_tests/models/account_statement_view.sql) at all (since it already explicitly tests the `true` method).

Whereas, I **would recommend changing** [account_view](https://github.com/fivetran/dbt_salesforce_formula_utils/blob/main/integration_tests/models/account_view.sql) as it is not currently testing for `false`, but is by default testing for `true` since the `full_statement_version` is not being defined as an argument. So you would update the acount_view model to be something like the following:

```sql
{{ salesforce_formula_utils.sfdc_formula_view(
    source_table='account',
    full_statement_version=false) 
}}
```

Then do the same to the other models that are not already defining the `full_statement_version` argument within the integration tests models.

Reply:
Oh I see, I misinterpreted. Let me make those changes and test based off of `full_statement_version=false`